### ISSUE TITLE
not reading AWS SSO credentials

### ISSUE THREAD

**Issue by saiyen2002** [one2nc/cloudlens#13]
I am not using IAM access key and secret for AWS, but AWS SSO. This means that there is no `~/.aws/credentials` file.

I see the following error

```
❯ cloudlens
panic: failed to read profiles -- open /Users/athiqur.rahman/.aws/credentials: no such file or directory

goroutine 1 [running]:
github.com/one2nc/cloudlens/cmd.readAndValidateProfile()
        github.com/one2nc/cloudlens/cmd/root.go:90 +0x2c5
github.com/one2nc/cloudlens/cmd.run(0x22f4ce0?, {0x1a280ec?, 0x0?, 0x0?})
        github.com/one2nc/cloudlens/cmd/root.go:63 +0x375
github.com/spf13/cobra.(*Command).execute(0x22f4ce0, {0xc000034240, 0x0, 0x0})
        github.com/spf13/cobra@v1.6.1/command.go:920 +0x847
github.com/spf13/cobra.(*Command).ExecuteC(0x22f4ce0)
        github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.6.1/command.go:968
github.com/one2nc/cloudlens/cmd.Execute()
        github.com/one2nc/cloudlens/cmd/root.go:41 +0x25
main.main()
        github.com/one2nc/cloudlens/main.go:8 +0x17
```

Let me know if you require further information

**Comment by chinmaysomani07** [comment]
Thanks for addressing this issue, we have solved it and SSO credentials will be supported in the next release.

**Comment by chinmaysomani07** [comment]
We have released the new `v0.1.2`
